blog_posts/6-tips-for-your-next-(remote)-onboarding.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 6 tips for your next (remote) onboarding
+type: list
+tags: career
+authors: fejes713
+cover: blog_images/golden-gate-bridge.jpg
+excerpt: Many developers find onboarding experience worrisome. Here are 6 helpful tips to make your onboarding experience a pleasant one.
+---
+
+Many developers find onboarding experience worrisome. Here are 6 helpful tips to make your onboarding experience a pleasant one:
+
+1. Ask for a project-specific tech stack and look for stuff that you might not be familiar with. One month is enough to cover the basics of anything you didn't learn at University and will save you lots of time (and stress) once the internship starts.
+
+2. Don't be embarrassed about asking for help. To get where they are now, your teammates had to go through the onboarding process and stuff you're facing now. Define with your team when you should ask for help. You'll run into obstacles every day. Some might take 30 minutes to solve, and others might take forever. Find a balance - You don't want to interrupt the whole team every 30 minutes, but don't let one problem turn your internship into an unpleasant experience.
+
+3. Most likely, you won't be micromanaged, but it's a good idea to keep everybody updated on your progress. Doing this, your team might offer some useful advice based on your progress or prepare you for upcoming obstacles in the task. If something isn't right, don't be silent. Your teammates aren't wizards, and they can't read your mind. If something is bothering you, you're stuck or not comfortable doing something. Just talk. Everybody is willing to help, but you need to let them know first.
```

Comment:
```suggestion
3. Most likely, you won't be micromanaged, but it's a good idea to keep everybody updated on your progress. Doing this, your team might offer some useful advice based on your progress or prepare you for upcoming obstacles in the task. If something isn't right, don't be silent. Your teammates aren't wizards, and they can't read your mind. If something is bothering you, you're stuck or not comfortable doing something - Just talk! Everybody is willing to help, but you need to let them know first.
```
Very minor suggestions, but I think we should emphasize the `Just talk` part as it is in the middle of the poin and could be missed more easily, exclamation mark to the rescue.